api/v1alpha1/ratelimit_types.go

```diff
@@ -138,6 +138,12 @@ type SourceMatch struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=256
 	Value string `json:"value"`
+
+	// InvertMatch specifies whether the value match result will be inverted.
+	//
+	// +optional
+	// +kubebuilder:default=false
```

Comment:
need a `// +notImplementedHide` comment, and remove it once it has been implemented